include/deal.II/base/array_view.h

```diff
@@ -773,107 +771,6 @@ make_array_view(ArrayView<Number, MemorySpaceType> &array_view)
 
 
 
-/**
- * Create a view to an entire Tensor object. This is equivalent to initializing
- * an ArrayView object with a pointer to the first element and the size of the
- * given argument.
- *
- * This function is used for @p const references to objects of Tensor type
- * because they contain immutable elements. Consequently, the return type of
- * this function is a view to a set of @p const objects.
- *
- * @param[in] tensor The Tensor for which we want to have an array view
- * object. The array view corresponds to the <em>entire</em> object but the
- * order in which the entries are presented in the array is an implementation
- * detail and should not be relied upon.
- *
- * @relatesalso ArrayView
- */
-template <int rank, int dim, typename Number>
-inline ArrayView<const Number>
-make_array_view(const Tensor<rank, dim, Number> &tensor)
```

Comment:
I think that we still want to deprecate these functions, simply because `Tensor` is not an array. You won't be able to create an array view to something that isn't an array.